components/Feature/SharePlan/index.module.scss

```diff
@@ -17,10 +17,32 @@
   }
 }
 
+.share-plan__collaborators-list {
+  @media (max-width: #{$screen-sm-min}) {
+    width: 100%;
+  }
+}
+.share-plan__collaborators-list-button,
 .share-plan__collaborators-list {
   width: 12rem;
+  border-bottom: none;
+
+  @media (max-width: #{$screen-sm-min}) {
+    float: left;
+    clear: left;
+  }
```

Comment:
Let's go through this tomorrow, we shouldn't be needing to add floats etc with the toolkit